translate/google/cloud/translate_v2/client.py

```diff
@@ -189,8 +189,8 @@ def translate(self, values, target_language=None, format_=None,
                                   in the query.
 
         :type model: str
-        :param model: (Optional) The model used to translate the text. The
-                      only accepted values are :attr:`BASE` and :attr:`NMT`.
+        :param model: (Optional) The model used to translate the text, such
+                      as ``'base'`` or ``'nmt'``.
```

Comment:
They are specified as a string in the API, and they do not want to update constants in the library whenever they add new ones.